viewing_party/party.py

```diff
@@ -1,23 +1,138 @@
+from statistics import mean, mode
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
-
+# -----------------------------------------
+# If valid, creates a movie with title, genre, and rating as values of a dict
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        return None
+    else:
+        movie = {"title" : title, "genre" : genre, "rating" : rating}
+    return movie
+
+# Adds movie dictionary to watched
+def add_to_watched(user_data, movie):
+    user_data = {}
+    user_data["watched"] = [movie]
+    return user_data

```

Comment:
This function is actually recreating the `user_data` dictionary and then re-assigns the `watched` key to a new list of one movie every time this function is called. To avoid this, the logic would be very similar to `add_to_watchlist`. 

```python
def add_to_watched(user_data, movie):
    user_data["watched"].append(movie)
    return user_data
````